### PR TITLE
Add local development instructions

### PR DESCRIPTION
**Pull request by burkemw3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,20 @@ wallabag app is a free and open source project developed by volunteers. Any cont
  * [Report bugs and make suggestions.](https://github.com/wallabag/ios-app/issues)
  * Write some code. Please follow the code style used in the project to make a review process faster.
 
+### Local Development
+
+Requirements
+
+* Xcode
+* [CocoaPods](https://github.com/CocoaPods/CocoaPods): `sudo gem install cocoapods`
```
Comment (bourvill) [inline README.md]: We use local Gemfile
prefer ``bundle installl``

File: README.md [hunk 2]
```diff
@@ -44,6 +44,20 @@ wallabag app is a free and open source project developed by volunteers. Any cont
...
+
+Steps
+
+0. In project directory, run `pod install`
```
Comment (bourvill) [inline README.md]: ``bundle exec pod install``

File: README.md (hunk 1 above)
Comment (burkemw3) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (burkemw3) [reply README.md]: Done

**Comment by bourvill** [comment]
Thank for your work

**Pull request merged**